source/docs/hardware/kop-item-resources/index.rst

```diff
@@ -0,0 +1,67 @@
+Kit of Parts Resources
```

Comment:
Each sub-section should have it's own toc

EG: mechanical/index

This is so it appears in the navigation bar cleanly. This toc can have a maxdepth of 2.